src/rospack.cpp

```diff
@@ -1648,6 +1650,7 @@ Rosstackage::computeDepsInternal(Stackage* stackage, bool ignore_errors, const s
         computeDeps(dep, ignore_errors, ignore_missing);
       }
     }
+    dep_node = dep_node->NextSiblingElement(depend_tag.c_str());
```

Comment:
In line 1631 `continue` is being used which won't execute this line and therefore end up in an infinite loop.